docs/quickstart.md

```diff
@@ -243,6 +236,72 @@ This will generate an ISO on the current directory with the name `elemental-<tim
 </TabItem>
 </Tabs>
 
+
+Note: Much but not all of the configuration provided in a `MachineRegistration` will be read at installation time instead of baked into the installation media. This means that many changes will not require using a new ISO.
+
+
+## Boot your nodes
+You can now boot your nodes with this newly created ISO, and they will:
+
+ - Register with the registrationURL given and create a per-machine `MachineInventory`
+ - Install Elemental Teal to the given device
+ - Shutdown
+ - Call home to your Rancher and wait
+
+After a few minutes you will see your nodes show up in the `MachineInventory` list.
+
+To watch this happen, you can run: 
+```
+kubectl get machineinventory -n fleet-default -w
+```
+
+
+## Build a Cluster
+
+Lastly, we need to build clusters out of our nodes! 
+
+This step can be done at any time (including before the machines were installed)
+
+We need two resources per cluster: a `Cluster` resource and a `MachineInventorySelectorTemplate` to know which machines are contained in the cluster.
+
+This selector is a simple matcher based on labels set in the `MachineInventory`, so if your selector is matching the `cluster-id` key with a value `myId` 
```

Comment:
Using labels is one possibility for the matcher. There are other matchers available.

Reply:
Added a note on where to find more options